.azure-pipelines/azure-pipelines-linux.yml

```diff
@@ -5,6 +5,20 @@ jobs:
     vmImage: ubuntu-latest
   timeoutInMinutes: 360
   steps:
+  - script: |
+      sudo mkdir -p /opt/empty_dir
+      for d in /opt/hostedtoolcache \
+          /usr/lib/jvm \
+          /usr/local/.ghcup \
+          /usr/local/android \
+          /usr/local/powershell \
+          /usr/share/dotnet \
+          /usr/share/swift \
```

Comment:
```suggestion
          /usr/share/swift \
          /opt/ghc \
```

Reply:
Yeah Billy noted [in the `conda-smithy` PR]( https://github.com/conda-forge/conda-smithy/pull/1747#issuecomment-1646207903 ):

> The `/opt/ghc` directory no longer seems to exist so I removed that.

That said, there would be no harm in keeping the directory (since we ignore any no-ops when removing)

Reply:
Done in commit ( https://github.com/conda-forge/staged-recipes/pull/23437/commits/e4f3f508404b6cee0bbe2f4daab374072f2f3366 )